tools/runners/run-test-suite.sh

```diff
@@ -112,35 +112,31 @@ do
     if [ "$IS_SNAPSHOT" == true ]
     then
         # Testing snapshot
-
         SNAPSHOT_TEMP=`mktemp snapshot-out.XXXXXXXXXX`
 
-        echo -n "[$tested/$total] ${ENGINE#$ROOT_DIR} $ENGINE_ARGS "
-        echo -n "--save-snapshot-for-global $SNAPSHOT_TEMP ${full_test#$ROOT_DIR}: "
-
+        cmd_line="${ENGINE#$ROOT_DIR} $ENGINE_ARGS --save-snapshot-for-global $SNAPSHOT_TEMP ${full_test#$ROOT_DIR}"
         ( ulimit -t $TIMEOUT; $ENGINE $ENGINE_ARGS --save-snapshot-for-global $SNAPSHOT_TEMP $full_test &> $ENGINE_TEMP )
```

Comment:
We cannot easily combine `cmd_line` and the actual execution since (after a recent change) we don't print exactly what we execute but strip some potentially lengthy prefix from the paths. (Otherwise, log would be too long and Travis would kill the process for exceeding log size limits.) That's why we have `${ENGINE#$ROOT_DIR}` and `${full_test#$ROOT_DIR}` above, where `$ROOT_DIR` is the prefix we don't print.